index.js

```diff
@@ -28,7 +28,16 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+
+function getAllMovieTitles(movies) {
+  let filmTitles = [];
```

Comment:
I might call this `movieTitles`. Nitpicky, I know, I know, but if we're talking convention here its probably what's best.